frontend/src/pages/Profile/UserSkills/UserSkills.tsx

```diff
@@ -68,21 +68,23 @@ const UserSkills: React.FC<Props> = ({
       <Divider />
     </SearchWrapper>
     <SkillsList>
-      {userSkills?.length ? (
-        userSkills.map((skill) => (
-          <div>
+      {userSkills && userSkills.size ? (
+        [...userSkills.keys()].map((group) => (
+          <div key={group}>
             <SkillsListItem>
               <SkillTitle>
-                <Star alt="skill" src={isSkillCompleted(skill) ? starContained : starEmpty} />
-                <Title>{skill.skillGroup}</Title>
+                <Title>{group || 'No group'}</Title>
               </SkillTitle>
               <SkillsInfoList>
-                {skill.skillList.map((skillItem) => (
-                  <SkillInfoContainer skillItem={skillItem} />
+                {userSkills.get(group)?.map((skill) => (
+                  <React.Fragment key={skill.name}>
```

Comment:
Something went wrong with stars icons https://prnt.sc/26xqit7

Reply:
i haven't prepared this PR yet, and you're getting another skill-data because i haven't updated front-database also